### PR TITLE
Added support for container stop

### PR DESCRIPTION
**Pull request by TeckniX**
This is my first time working with Electron/React/etc and couldn't figure out where the test units were. I believe that I added the proper 'Stop' container functionality, but I could also be very wrong.

Any input is appreciated!

### CODE REVIEW

**Comment by Elesant** [comment]
Thanks @TeckniX! This is awesome! You even added the nice icons haha :) I'm sorry the test units aren't complete yet. I will take a look and play around with this PR.

**Comment by TeckniX** [comment]
That would explain it ;) - Yea the little icon was a quick photoshop deal. I wasn't sure who was responsible for the layout design. 

As previously mentioned quite fresh on this type of build, is there an easy way to debug/log actions/events, so that I can follow clicks etc?

File: src/components/ContainerDetailsSubheader.react.js [hunk 1]
```diff
@@ -54,6 +54,12 @@ var ContainerDetailsSubheader = React.createClass({
     }
     return (this.props.container.State.Downloading || this.props.container.State.Restarting);
   },
+  disableStop: function () {
```
Comment (jeffdm) [inline src/components/ContainerDetailsSubheader.react.js]: rename to stopDisabled

**Comment by jeffdm** [comment]
This feature is in HUGE demand. Awesome. Added one comment, otherwise LGTM.

Thanks so much for taking the time to make this PR. One last thing, can you [sign your commit](https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work)?

**Comment by TeckniX** [comment]
Can I just push a blank update and add as the comment my 'Signed-off-by: Joe Smith <joe.smith@email.com>' ?

**Comment by jeffdm** [comment]
This should work:

`git commit -s --amend`
`git push -f origin master`

We're working making signing automatic / easier for contributors. Sorry about that!

**Comment by jeffdm** [comment]
Just took this PR for a spin. Fantastic! Looks great too. One last thing: center the text and then this is ready to merge:

screen shot 2015-04-24 at 6 18 31 pm

**Comment by TeckniX** [comment]
Good catch - label should be centered now - Thanks!

**Comment by jeffdm** [comment]
Great! Merging. Thanks for the contribution, @TeckniX

**Comment by TeckniX** [comment]
Glad I could help :dancer:

**Pull request merged**